sonota.py

```diff
@@ -474,10 +474,21 @@ def promptforval(msg):
         if val:
             return val
 
+def resource_path(relative_path):
+    """ Get absolute path to resource, works for dev and for PyInstaller """
+    try:
+        # PyInstaller creates a temp folder and stores path in _MEIPASS
+        base_path = sys._MEIPASS
+    except Exception:
+        base_path = os.path.abspath(".")
```

Comment:
Thinking about this one, using this should actually make it more reliable for all situations:
`os.path.dirname(sys.argv[0])`
(Although I'm not sure if it'll work for the exe, but even if we just for for this line, it would be ideal I think).

Reply:
but i dont get this :S
you want to set the path by argument? but u cannot change the path, cuz the path has to show to the static folder for the bin files in all situations.

Reply:
`sys.argv[0]` in the path to the current running program. So it's a way to find where all of the other directories etc should be (rather than relying on `.`), does that make sense?